test/parallel/test-abort-uncaught-exception.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const node = process.execPath;
+
+if (process.argv[2] === 'child') {
+  throw new Error('child error');
+} else {
+  run('', null);
+  run('--abort-on-uncaught-exception', 'SIGABRT');
```

Comment:
That'll be a problem for a lot of devs, like myself. A test is nice, but allowing them to abort isn't so nice.